### ISSUE TITLE
Wasm module not found 404 for <istio_version>=10

### ISSUE THREAD

**Issue by noman1897** [istio/istio.io#10094]
Hi guys,

while trying the k8s artefacts in the  **Distributing WebAssembly Modules (Experimental)** functionality (https://istio.io/latest/docs/ops/configuration/extensibility/wasm-module-distribution/) i had the error 404  in the istio-ingressgw logs when the envoy filter go and try to download it with using this url " **https://github.com/istio-ecosystem/wasm-extensions/releases/download/1.10.0/basic-auth.wasm** ".

Looking at the document in this **istio.io** repo the file : istio.io/content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md we use the variable istio_version, so i would like to know what could be the best way to make the change because changing that url in the basic-auth-config yaml  with :
                 uri: https://github.com/istio-ecosystem/wasm-extensions/releases/download/1.9.2/basic-auth.wasm

worked, but i don't think it's a good idea to hard code the previous istio release version in the latest documentation.
What do you think we can do to correct it?

My istio version :
control plane version: 1.9.5
data plane version: 1.9.5 (4 proxies)

thanks!

**Comment by ericvn** [comment]
@bianpengyuan I think we discussed this earlier. What is the process to make sure we have a 1.10.0 and 1.11.0 (coming soon) release available for the docs.

**Comment by bianpengyuan** [comment]
Yeah, Sorry, have not got time to automate release in that repo. I will need to manually create a release build. Will do it in this week and try to automate it as well.

**Comment by ericvn** [comment]
The link specified in the documents is generated from https://github.com/istio-ecosystem/wasm-extensions/releases/download/{{< istio_version >}}.0/basic-auth.wasm where istio_version is 1.10, 1.9, etc. at the time this page was done, there was discussion that there would always be a .0 so the author coded it up that way. 

I don't think the file necessarily matches the actual version of Istio, so there isn't a 1.9.6 for example.

The page author may want to include something in the update_doc_ref script where it finds the latest release and sed's that into the document. I'm not sure what happens if the minor version is different. For example, for Istio 1.10, is it appropriate to use the latest 1.9 extension as the 1.10.0 doesn't exist?

**Comment by ericvn** [comment]
As an aside, if the Issue creator looked at the 1.9 version of the documentation, https://istio.io/v1.9/docs/ops/configuration/extensibility/wasm-module-distribution/, it does have 1.9.2.
